whois-commons/src/test/resources/whois.properties

```diff
@@ -19,6 +19,9 @@ dir.grs.import.download=var${jvmId:}/grs
 # API configuration
 api.rest.baseurl=http://rest.db.ripe.net
 
+# API Client Certificate Authentication
+api.client.auth.baseurl=https://rest-cert-dev.db.ripe.net
```

Comment:
This is an optional property, I suggest only setting it when `client.cert.auth.enabled` is true (i.e. in `AbstractClientCertificateIntegrationTest.java`)